packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -4,28 +4,51 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { Aliases, AuthInfo } from '@salesforce/core';
-import { ForceConfigGet } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
-
+import {
+  Aliases,
+  AuthInfo,
+  ConfigAggregator,
+  ConfigFile
+} from '@salesforce/core';
+import * as path from 'path';
+import { getRootWorkspacePath, hasRootWorkspace } from './index';
 export class OrgAuthInfo {
   public static async getDefaultUsernameOrAlias(
```

Comment:
Both here and on `getDefaultDevHubUsernameOrAlias` we need to account for the possibility of the local configuration not being in place or incomplete. For example, when you create a new project we do not create a `.sfdx-config.json` file. It's until the user created a scratch org or auths to an org that this gets created.
If the file is present, then there's also a possibility of `defaultusername` and `defaultdevhubusername` not being present.

Reply:
That's a good point, I'll add in checks for the file not being present and usernames not being present as well.